### ISSUE TITLE
ISO Level on DFP and DFRP

### ISSUE THREAD

**Issue by AlexBLCKFNG** [ricohapi/theta-plugins#11]
Hi there,

I´m Alex, just joined this plattform to adress the following topic:

I used both the Dual Fisheye Plugin and the newer Raw version and discovered that I get slightly less noise in my bracketed DNG-shots with the original DFP, probably because the original DFP shoots with an ISO of 64 whereas the offset for the DFRP is ISO 80.

Why is this difference in ISO-offset and is there any chance to get the Raw Plugin to shoot in ISO 64 as well? Or maybe through a future update?

Thanks for your input 
Alex

**Comment by DaisukeHohjoh** [comment]
Hi,

Actually, ISO 64 is not supported by Theta Z1 officially like this. https://github.com/ricohapi/theta-api-specs/blob/main/ricoh-theta-plugin/camera-api.md#iso-sensitivity

I guess that DFP sets ISO 64 in their plugin, which should be rejected by Z1 firmware originally. If my guess is correct, it is the bug both for DFP and Z1 firmware. DFP is not developed by Ricoh, but anyway I will check what will be caused when plugin sets ISO 64 .

Best regards,
Daisuke Hohjoh

**Comment by AlexBLCKFNG** [comment]
Thanks for your reply. That´s just what I encountered when showing the image information in Lightroom. For DFP it says ISO 64 and for DFRP ISO 80. And there´s definitely a little bit less noise in the DFP-picture compared to the DFRP-picture.

**Comment by DaisukeHohjoh** [comment]
Hi,

I verified that plugin can set ISO 64 (also ISO 50) but it should be wrong behavior originally, because Z1 image sensor cannot guarantee ISO 64 with considering the variation in sensitibity. As you said, when plugin sets ISO 64 by Camera API, image noise will be reduced a little but it should not be truly ISO 64.

I think that this bug should be fixed by Z1 firmware side, so that will fix this phenomenon in next Z1 firmware version. After fixing this, DFP will capture ISO 80 same as DFRP.

Best regards,
Daisuke Hohjoh

**Comment by AlexBLCKFNG** [comment]
Ok, that most likely solves the mystery. So both plugins will probably shoot with ISO 80 in the near future...!?

**Comment by DaisukeHohjoh** [comment]
Yes, after next Z1 firmware released, both plugin will shoot ISO 80. Because new firmware will overwrite ISO setting to ISO 80 when plugin tends to set ISO 64.

**Comment by AlexBLCKFNG** [comment]
Ok, thanks for contributing 👍🏼

**Comment by DaisukeHohjoh** [comment]
Dear @AlexBLCKFNG ,

As there hasn't been any activity on this issue for a prolonged period, we will be closing it for now.
Should you have further questions or if the issue persists, please don't hesitate to reopen or comment.

THETA Z1 firmware v3.20.3 applied the fix, in May 2024.

Best Regards,
Daisuke Hohjoh
